scripts/generate_sudo_list.py

```diff
@@ -48,7 +48,7 @@ def compare_sudo_cmds_lists(file_path):
         temp_file.write(generated_doc.encode())
         try:
             if filecmp.cmp(temp_file.name, file_path):
-                sys.stdout.write("The sudo command list document is up to date.\n")
+                pass
             else:
```

Comment:
replace `if <somethings>/pass/else` pattern with `if not <something>`

Reply:
done and rebased